src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -844,10 +844,10 @@ public int NodeSearchTagSizeLimit
 
         /// <summary>
         /// The Version of the IronPython package that Dynamo will download when it is found as missing in graphs.
-        /// This static property is not serialized and is assigned IronPythonResolveTargetVersion's value
+        /// This property is not serialized and is assigned IronPythonResolveTargetVersion's value
         /// if found at deserialize time.
         /// </summary>
-        internal static Version ironPythonResolveTargetVersion = new Version(2, 4, 0);
+        internal Version ironPythonResolveTargetVersion = new Version(3, 0, 0);
```

Comment:
And you don't want to leave this static because ...

Reply:
if it's static when the migrator loads the old preferences it modifies the new preferences and we never even have a chance of looking up the old value.

**I don't think it makes sense for any preferences to backed by static fields.**

Reply:
thanks, it was already easily available in the extension where it was needed AFAIK.

Reply:
But yes not having to read properties from the file and then deserialize this makes sence.